### PR TITLE
[IMP] web: support of array_agg in mockReadGroup

### PR DESCRIPTION
**Pull request by Polymorphe57**

### CODE REVIEW

File: addons/web/static/tests/helpers/mock_server.js [hunk 1]
```diff
@@ -1023,29 +1026,36 @@ export class MockServer {
...
+                        group[name] = 0;
+                        for (const r of records) {
+                            group[name] += r[fieldName];
+                        }
+                        break;
+                    }
+                    case "many2one": {
+                        const ids = records.map((r) => r[fieldName]);
+                        const uniqueIds = [...new Set(ids)];
+                        if (func === "array_agg") {
```
Comment (Polymorphe57) [inline addons/web/static/tests/helpers/mock_server.js:1052]: remove false from uniqueIds?

Comment (Polymorphe57) [reply addons/web/static/tests/helpers/mock_server.js:1052]: apparently array_agg does not give uniqueness

**Comment by aab-odoo** [comment]
robodoo rebase-ff r+

**Review by aab-odoo — APPROVED** [review]
robodoo rebase-ff r+

**Pull request closed without merging**